CMakeLists.txt

```diff
@@ -494,7 +497,7 @@ endforeach()
 target_include_directories(${NVFUSER_CODEGEN} PRIVATE "${CMAKE_BINARY_DIR}/include")
 
 # -- install nvfuser cmake config files and symlink to build binaries
-install(EXPORT NvfuserTargets FILE NvfuserConfig.cmake DESTINATION share/cmake/nvfuser)
+install(EXPORT NvfuserTargets FILE NvfuserConfig.cmake DESTINATION cmake)
```

Comment:
this probably will mess up where pytorch's structure on cmake configs? they expects `torch/share/cmake/nvfuser`, rather than `torch/cmake`.

Reply:
Is this for building `pytorch` with `top_level_nvfuser`? The `NvfuserConfig.cmake` should be discovered with `import nvfuser; print(nvfuser.cmake_prefix_path)`

Reply:
You need this for both build methods. So I'm wondering what happens with submodule build?

Eve with submodule build, nvfuser is a separate library that needs cmake config in order to be searched. (The difference is that the file is under `torch/share/cmake/nvfuser`, which is already added to cmake search path along with torch.utils).